distribution/scripting.md

```diff
@@ -18,6 +18,8 @@ Local scripts can **not** alter the game state. This allows each player to enabl
 
 Remote scripts on the other hand can alter the game state in certain contexts, thus must be enabled for every player in a multiplayer game. Players **cannot** enable or disable remote scripts for multiplayer servers they join. Instead the server will upload any remote scripts that have been enabled on the server to each player. This allows servers to enable scripts without players needing to manually download or enable the same script on their end.
 
+The authors should also define a valid license for the plugin, making it clear to the community whether that plugin can be altered, copied, etc. OpenRCT2 supports all licenses listed on [choosealicense](https://choosealicense.com/appendix/), make sure you pick one of them and use its identifier, as listed on [SPDX](https://spdx.org/licenses/).
```

Comment:
OpenRCT2 does not "support" any licence.

If we are to _suggest_ a licence to people who don't know yet, make it MIT, I say - it's very common and allows others do as they please, including linking to GPL-incompatible software.

Reply:
I changed it to `MIT` in he example and rewrote this section slightly.